flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -158,11 +156,6 @@ def serialize_all(
             )
             old_style_entities.append(o)
 
-        # PythonInstanceTasks will not be picked up by the above, so we need to reiterate
-        for o, v in load_module_object_for_type(pkgs, PythonInstanceTask, additional_path=local_source_root).items():
```

Comment:
just to double check, we don't need this for https://github.com/flyteorg/flytekit/pull/391?

Reply:
nope, getting rid of it, using the instance tracking metaclass now.